tests/driver/api.test.ts

```diff
@@ -53,8 +53,11 @@ describe('Mongoose Model API level tests', async () => {
         dbUri = testClient.uri;
         isAstra = testClient.isAstra;
     });
-    let mongooseInstance: Mongoose | null = null;
```

Comment:
`Mongoose ` import on line 22 is not used and getting a warning message. Can we remove that please?